docs/source/en/api/pipelines/text_to_video_zero.mdx

```diff
@@ -0,0 +1,154 @@
+
+
+## Usage example
+
+### Text-To-Video
+
+To generate a video from prompt, run the following python command
+```python
+import torch
+from diffusers import TextToVideoZeroPipeline
+
+model_id = 'runwayml/stable-diffusion-v1-5'
+pipe = TextToVideoZeroPipeline.from_pretrained(model_id, torch_dtype=torch.float16).to('cuda')
+
+prompt = "A panda is playing guitar on times square"
+result = pipe(prompt=prompt).images
+imageio.mimsave('video.mp4', result, fps=4)
+```
+You can change these parameters in the pipeline call:
+* Motion field strength (see the [paper](https://arxiv.org/abs/2303.13439), Sect. 3.3.1):
+    * `motion_field_strength_x` and `motion_field_strength_y`. Default: `motion_field_strength_x=12`, `motion_field_strength_y=12`
+* `T` and `T'` (see the [paper](https://arxiv.org/abs/2303.13439), Sect. 3.3.1)
+    * `t0` and `t1` in the range `{0, ..., num_inference_steps}`. Default: `t0=45`, `t1=48`
+* Video length:
+    * `video_length`, the number of frames video_length to be generated. Default: `video_length=8`
+
+
+### Text-To-Video with Pose Control
+To generate a video from prompt with additional pose control, follow these steps
+
+Read video containing extracted pose images from path
+```python
+import imageio
+
+reader = imageio.get_reader('path/to/your/pose/video',  'ffmpeg')
+frame_count = 8
+pose_images = [Image.fromarray(reader.get_data(i)) for i in range(frame_count)]
+```
+To extract pose from actual video, read [ControlNet documentation](./stable_diffusion/controlnet).
+
+Run `StableDiffusionControlNetPipeline` with our custom attention processor
+
+```python
+import torch
+from diffusers import StableDiffusionControlNetPipeline, ControlNetModel
+from diffusers.pipelines.text_to_video_synthesis.pipeline_text_to_video_zero import CrossFrameAttnProcessor
+
+model_id = 'runwayml/stable-diffusion-v1-5'
+controlnet = ControlNetModel.from_pretrained("lllyasviel/sd-controlnet-openpose", torch_dtype=torch.float16)
+pipe = StableDiffusionControlNetPipeline.from_pretrained(model_id, controlnet=controlnet, torch_dtype=torch.float16).to('cuda')
+pipe.unet.set_attn_processor(CrossFrameAttnProcessor(batch_size=2))
+
+prompt = 'Darth Vader dancing in a desert'
+result = pipe(prompt=[prompt] * len(pose_images), image=pose_images).images
+imageio.mimsave('video.mp4', result, fps=4)
+```
+
+
+### Text-To-Video with Edge Control
+To generate a video from prompt with additional pose control, follow the steps described above for pose-guided generation using canny edge ControlNet model <br/>
+
+
+### Video Instruct-Pix2Pix
+To perform text-guided video editing, run the following python command:
+
+Read video from path
+```python
+import imageio
+
+reader = imageio.get_reader('path/to/your/video',  'ffmpeg')
+frame_count = 8
+video = [Image.fromarray(reader.get_data(i)) for i in range(frame_count)]
```

Comment:
Same as what I mentioned in https://github.com/huggingface/diffusers/pull/2954/files#r1158020490.

Reply:
Done!